src/general/SpeciesInfo.cs

```diff
@@ -0,0 +1,11 @@
+/// <summary>
+///   Class that stores species-wide information.
+///   It can be expanded to have any species parameter we might want to keep,
+///   e.g. base structure, behavioral values... and such to draw species history.
```

Comment:
These two lines feel like a remark to me.
Btw remarks should be formatted like this:
```
27:    /// <remarks>
28-    ///   <para>
29-    ///     There's a small bug where if the tutorials are turned back on and then displayed, that leaves the checkboxes
30-    ///     unchecked, when things become visible all the enable tutorials check boxes would need to read this value
31-    ///     to fix this
32-    ///   </para>
33:    /// </remarks>
```